src/constants/translations/en/common.json

```diff
@@ -69,4 +69,4 @@
     "spanish": "Spanish",
     "arabic": "Arabic"
   }
-}
+}
```

Comment:
why have you pushed this file, if you changed nothing in it

Reply:
lint added required space for this file, so, I see no reason to revert changes in this file now